src/components/typography/typography.html

```diff
@@ -0,0 +1,197 @@
+                    </a>
+                    <a href="" class="social__links">
+                        <i class='bx bxl-github' ></i>
+                    </a>
+                </div>
+			</header>
+			<nav class="nav flex-col-align-center">
+				<h1 class="h4 nav__head">Get Started</h1>
+				<ul class="nav__items flex-col-align-center">
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/usage/usage.html">Usage</a>
+					</li>
+					<li class="nav__list"><a class="nav__links" href="#">Colors</a></li>
+					<li class="nav__list"><a class="nav__links nav_link--selected" href="/src/components/typography/typography.html">Typography</a></li>
+				</ul>
+				<h1 class="h4 nav__head">Components</h1>
+				<ul class="nav__items flex-col-align-center">
+					<li class="nav__list">
+						<a
+							class="nav__links"
+							href="/src/components/avatar/avatar.html"
+							>Avatar</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/badge/badge.html"
+							>Badge</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/alert/alert.html"
+							>Alert</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/button/button.html"
+							>Button</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/card/card.html">Card</a>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/input/input.html"
+							>Input</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/image/image.html"
+							>Image</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/snackbar/snackbar.html"
+							>Snackbar</a
+						>
+					</li>
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/lists/lists.html"
+							>Lists</a
+						>
+					</li>
+				</ul>
+			</nav>
+	
+			<!-- Main Start -->
+	
+			<main class="main">
+				<!-- ----Typography----- -->
+	
+				<h2 class="h2 m_doc__heading">Typography</h2>
+				<p class="m_doc__para">
+                    Typography component makes it easy to apply a default set of font weights and sizes in your application.
+					Use typography to present your design and content as clearly and efficiently as possible. 
+				</p>
+	
+				<!-- SNIPPET CONTAINER -->
+	
+				<div class="snippet_container">
+					<script
+						type="text/plain"
+						class="line-numbers language-css snippet_container__script"
+					>
+						// CSS
+						@import url("https://magnificentui.netlify.app/src/components/typography/typography.css");
+	
+						// HTML
+						<link rel="stylesheet" href="https://magnificentui.netlify.app/src/components/typography/typography.css">
+					</script>
+				</div>
+	
+				<!-- Headings component -->
+				<h3>Headings</h3>
+	
```

Comment:
try to reduce these line gaps

Reply:
Cool! I have kept them deliberately to make them more readable. I will trim them after completing the project.

Reply:
ok